client/src/tokens/typography.stories.tsx

```diff
@@ -43,6 +50,16 @@ export const TypeScale = () => (
           </td>
         </tr>
       ))}
+      {Object.entries(fontFamily).map(([key, , cls = `w-font-${key}`]) => (
+        <tr key={key}>
+          <td>
+            <span className={cls}>{cls}</span>
+          </td>
+          <td>
+            <code>.{cls}</code>
+          </td>
+        </tr>
+      ))}
```

Comment:
I don’t get why we would want this here? 🤔  The text styles on here are meant to be readily applicable to any bit of text in the CMS, used as much as possible to guarantee UI consistency. The font family utilities are more low-level and should be used in much fewer cases (as little as possible).

Reply:
ok